.serverless_plugins/serverless-single-page-app-plugin.js

```diff
@@ -60,11 +60,10 @@ class ServerlessPlugin {
   // syncs the `app` directory to the provided bucket
   syncDirectory() {
     const s3Bucket = this.serverless.variables.service.custom.s3Bucket;
-    const buildFolder = this.serverless.variables.service.custom.client.distributionFolder;
     const args = [
       's3',
       'sync',
-      `${buildFolder}/`,
+      'app/',
```

Comment:
You should not hard code a path to your distribution folder. This should be set in `serverless.yml` file, see below.